### DISCUSSION TITLE
Automation Loop cannot doing filter in Query Rows by {{ loop.currentItem._id }}

### DISCUSSION THREAD

**Discussion by xnohat** [Budibase/budibase#6329] [Help]
Hi,

I got a problem when using Automation Looping feature
Steps:
1. Create Table A with anything
2. Create Table B and a Relationship column to Table A, create another Static Formula Column name table_a_id with formula {{ relationship_column_to_table_a.0._id }}
3. Create Automate:
4. Create Step 1: Query Rows table A
5. Create Step 2: Query Rows table B , add Looping array of {{ step.1.rows }}, Filter set to: *table_a_id equal binding {{ loop.currentItem._id }}
6. Test

Expect:
1. {{ loop.currentItem._id }} return _id of currentItem (current row of Table A rows )
2. Output of Query 2: rows have table_a_id match _id of rows in table A

Current Results:
1. {{ loop.currentItem._id }} return nothing in filter section
2. query 2 return all rows of table 2 without filtering

![image](https://user-images.githubusercontent.com/4613808/173803801-e1cc23f8-5a7f-475c-a2dc-124d3d9396e8.png)

I attached a sample app to demo this problem.

It's this bug or I using automation loop in wrong way ? any workaround for this use case ?

Many thanks,
[demoapp-export-1655289044680.txt](https://github.com/Budibase/budibase/files/8908471/demoapp-export-1655289044680.txt)

**Comment by melohagan** [comment]
Hi @xnohat 

Thanks for reporting. Filters for query rows using any binding seem to be broken at the moment: https://github.com/Budibase/budibase/issues/6049

**Reply by xnohat** [reply]
Thanks, I waiting for fix, this feature very important
